lib/pymedphys/_dev/docs.py

```diff
@@ -9,6 +9,12 @@
 
 
 def build_docs(args):
+    subprocess.check_call(
```

Comment:
this should have been done in a different PR.
Especially when a PR has many files involved, the PR should have a single intent.

Reply:
Yup, good call. See https://github.com/pymedphys/pymedphys/pull/1169.

If that gets merged first then it will no longer be a part of the changelog in this PR.